src/neofs_testlib/models/object.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class Object:
```

Comment:
python already have object class. 
Need to be renamed to StorageObject or something else

Reply:
Done